### ISSUE TITLE
Required files for running on browser

### ISSUE THREAD

**Issue by extend1994** [WebReflection/url-search-params#18]
Hi
I am a member of CDNJS and in charge of adding this library to our CDN.
Could you please tell me which files do we need to let the library run on browser well?
Is `url-search-params.js` enough?
Thanks! :smile:

**Comment by WebReflection** [comment]
The min version is the one with less chars:
https://github.com/WebReflection/url-search-params/blob/master/build/url-search-params.js

The max version is the explicit one:
https://github.com/WebReflection/url-search-params/blob/master/build/url-search-params.max.js

**Comment by WebReflection** [comment]
P.S. amd version and commonjs version there too:
https://github.com/WebReflection/url-search-params/tree/master/build

TL;DR the build folder is what you are "linking" for

**Comment by extend1994** [comment]
@WebReflection Thanks again!
